Doxyfile.in

```diff
@@ -0,0 +1,14 @@
+PROJECT_NAME           = "@CMAKE_PROJECT_NAME@"
+PROJECT_NUMBER         = @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@
+STRIP_FROM_PATH        = @PROJECT_SOURCE_DIR@ \
+                         @PROJECT_BINARY_DIR@
+INPUT                  = @doxy_main_page@ \
+                         @PROJECT_SOURCE_DIR@/src \

```

Comment:
I think that you don't want to include this one, we don't want the user to access the documentation of `.hpp` files in the src because they are private (like the `WindowImpl` class).
And, can you omit the namespace `priv` in that file? And also the `include/Ext` directory which contains external library?